quix-frontend/client/src/lib/sql-autocomplete/adapter/sql-autocomplete-adapter.ts

```diff
@@ -47,6 +48,25 @@ export class SqlAutocompleter implements IAutocompleter {
         return [];
     }
   }
+  public async getAllCompletionItemsFromQueryContextCollumn(queryContext: any) {
```

Comment:
Try to be consistent with your function naming, here the function names structure is get{Something}From{SomethingElse}

`getAllCompletionItemsFromQueryContextCollumn`
`getCompletionItemsFromQueryContext`

What are the differences between the return value of the 2? AllCompletionItems and CompletionItems, why in one place I'm getting ALL and in another place I'm not.

Try to think about new consistent names for it.